web/src/RootUser.jsx

```diff
@@ -32,21 +33,24 @@ const reducer = (state, action) => {
 
 const initialState = {
   rootPassword: null,
-  isFormOpen: false
+  isFormOpen: false,
+  SSHKey: ""
 };
 
 export default function RootUser() {
   const client = useInstallerClient();
   const [state, dispatch] = useReducer(reducer, initialState);
-  const { rootPassword, isFormOpen } = state;
+  const { rootPassword, isFormOpen, SSHKey } = state;
   const hiddenPassword = "_____DINSTALLALER_PASSWORD_SET";
 
   useEffect(async () => {
     const rootPassword = (await client.users.isRootPassword()) ? hiddenPassword : "";
+    const SSHKey = await client.users.rootSSHKey();
     dispatch({
       type: "LOAD",
       payload: {
-        rootPassword
+        rootPassword,
+        SSHKey
```

Comment:
I am not sure about capitalization here. It is not wrong, but I would go for something like `sshKey`.

Reply:
Well, I try to use consistently SSH key as acronym. So all big. If we go with lower form it needs to be changed on multiple places and e.g. `ssh` does not look bad, but e.g. `RootSshKey` looks really strange for me.